vhost-device-vsock/src/txbuf.rs

```diff
@@ -55,7 +55,11 @@ impl LocalTxBuf {
         // Check if there is more data to be wrapped around
         if len < data_buf.len() {
             let remain_txbuf = &mut self.buf[..(data_buf.len() - len)];
-            data_buf.copy_to(remain_txbuf);
+            // SAFETY ok to unwrap because len < data_buf.len()
+            data_buf
```

Comment:
can we use `VolatileSlice::read`? I mean something like `data_buf.read(remain_txbuf, len)`?

Reply:
Done.
Used `read_slice` instead since it's not necessary to deal with incomplete reads.